<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QWidget" name="Timer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #87ceeb;
}

QVBoxLayout, QHBoxLayout {
       margin: 0;
       padding: 0;
       spacing: 10px;
   }</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>180</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border:0px 0px 0px 0px;
    font-size: 25px;
    background-color:transparent;
    min-width:40px;
    max-width:40px;
    min-height:55px;
    max-height:55px;
    border-radius:15%;
}

QLineEdit:hover {
    border-width: 2px;
    border-color: black;
    border-style: solid;
}</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel {
    font-size: 20px;
    min-height: 50px;
    max-height: 50px;
    padding-bottom: 5px;
    min-width: 15px;
    max-width: 15px;
}</string>
           </property>
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border:0px 0px 0px 0px;
    font-size: 25px;
    background-color:transparent;
    min-width:40px;
    max-width:40px;
    min-height:55px;
    max-height:55px;
    border-radius:15%;
}

QLineEdit:hover {
    border-width: 2px;
    border-color: black;
    border-style: solid;
}</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel {
    font-size: 20px;
    min-height: 50px;
    max-height: 50px;
    padding-bottom: 5px;
    min-width: 15px;
    max-width: 15px;
}</string>
           </property>
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border:0px 0px 0px 0px;
    font-size: 25px;
    background-color:transparent;
    min-width:40px;
    max-width:40px;
    min-height:55px;
    max-height:55px;
    border-radius:15%;
}

QLineEdit:hover {
    border-width: 2px;
    border-color: black;
    border-style: solid;
}</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 16px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #45a049;
}

QPushButton:pressed {
    background-color: #3e8e41;
}</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 16px;
    margin: 5px;
}

QPushButton:hover {
    background-color: #e53935;
}

QPushButton:pressed {
    background-color: #d32f2f;
}</string>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
